Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2259,14 +2258,10 @@ private void HandleProximityScaling()
                 // Grab points within sphere of influence from valid pointers
                 foreach (var pointer in proximityPointers)
                 {
-                    if (IsPointWithinBounds(pointer.Position, maxRadius))
-                    {
```

Comment:
The one issue I see with removing this check is that we end up calculating the distance of a point to each of the scale handlers, whereas before we could potentially avoid this by only calculating the distance if it was near the bounding box object. However, since we don't actually know the scale of the object that is being manipulated, determining when the pointer is "near" the object becomes kind of tricky. There shouldn't be a large performance impact unless there are a lot of objects with their handlers visible at all the same time.

Reply:
if i remember correctly a performance issue was why troy put this in place - bounding box is heavily used and there can be a lot of handles accumulated in a scene - so removing this check might hit us at a later point

Reply:
can you verify that this doesn't have any performance implications on scenes with many bounding box / bounds control added ?

Reply:
With 128~ bounding boxes in the scene I go from 29 to around 24 fps

With 65~ ish bounding boxes I go from 35-37 fps to around 33

With 32~ ish bounding boxes its 102 with the check and 99 without